pkg/endpoint/endpoint.go

```diff
@@ -162,6 +174,16 @@ var (
 	EndpointOptionLibrary = option.OptionLibrary{
 		OptionAllowToHost: &OptionSpecAllowToHost,
 	}
+
+	// ciliumEPControllerLimit is the range of k8s versions with which we are
+	// willing to run the EndpointCRD controllers
+	ciliumEPControllerLimit, _ = go_version.NewConstraint(">= 1.7.0")
```

Comment:
Please write `> 1.6` I've found out that constrain doesn't work with `1.7.0-alpha`

Reply:
done.